src/clients/dotnet/TigerBeetle.Tests/IntegrationTests.cs

```diff
@@ -1130,8 +1132,34 @@ public TBServer()
             format.WaitForExit();
             if (format.ExitCode != 0) throw new InvalidOperationException("format failed");
 
-            process = Process.Start(TB_SERVER, START);
+            process = new Process();
+            process.StartInfo.FileName = TB_SERVER;
+            process.StartInfo.Arguments = START;
+            process.StartInfo.RedirectStandardError = true;
+            process.Start();
             if (process.WaitForExit(100)) throw new InvalidOperationException("Tigerbeetle server failed to start");
+
+            var readPortEvent = new AutoResetEvent(false);
+            var addressLogged = "";
+            var stderrLogged = new StringBuilder();
+            process.ErrorDataReceived += (sender, e) => {
+                var line = e.Data;

```

Comment:
Running `dotnet format`:

```suggestion
            process.ErrorDataReceived += (sender, e) =>
            {
                var line = e.Data;

```

We might add `dotnet format --verify-no-changes` to our CI.

Reply:
Good call, will add to https://github.com/tigerbeetle/tigerbeetle/pull/1201